README.md

```diff
@@ -1 +1,272 @@
+- initialize npm to create package.json file by running the command: 
+> $ npm i -y      
+
+(here i for install & -y for saying yes to all)
+
+*************
+For Webpack:
+*************
+- To install **Webpack and Webpack CLI**:
+> $ npm install webpack webpack-cli --save-dev
+
+- To optaionally install a JS library as an example (e.g. **Lodash**)
+> $ npm install --save lodash (optional)
+
+- To install the **HtmlWebpackPlugin** plugin.
+> $ npm install --save-dev html-webpack-plugin
+
+- To install **style-loader and css-loader** 
+> $ npm install --save-dev style-loader css-loader
+
+- To install the **webpack-dev-server** as a dev dependency (to use live reloading)
+> $ npm install --save-dev webpack-dev-server
+
+- List of the commands to run the build (with all the alternatives).
+
+To build and run with the webpack dev server:
+> $ npm start
+
+(Will be served at: http://localhost:8080/)
+
+- To build only :
+> $ npm run build
+
+(and then may be served using a local server e. g. Live Server in VS Code)
+
+- (Optional Alternatives)
+> $ npm run webpack or, $ npm run webpack --mode production
+
+> $ npx webpack
+
+> $ npx webpack --mode production
+
+> $ npx webpack --config webpack.config.js
+
+************************
+For Webhint (optional):
+************************
+- Run
+> npm install --save-dev hint@7.x
+- To fix validation errors run: 
+> npx hint .
+
+**************
+For Stylelint
+**************
+- Run 
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+- Copy **.stylelintrc.json** to the root directory of your project.
+
+- Run 
+> npx stylelint "**/*.{css,scss}" 
+on the root of your directory of your project.
+- Fix linter errors.
+
+*******
+ESLint
+*******
+- Run 
+> npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+- Copy **.eslintrc.json** to the root directory of your project.
+
+- Run 
+> npx eslint .  
+on the root of your directory of your project.
+- Fix linter errors.
+
+### Deployment
+You can deploy this project using:
+- GitHub Pages for the main branch. A deployed version can be found here:
+https://...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+> Developed by: 
+
+### Author
+- GitHub: [@porag-m06](https://github.com/porag-m06)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/muhammad-porag-nsu-cse/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Improve Aesthetics & Design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/porag-m06/Porag-s-Amazing-To-Do-List/issues).
+
```

Comment:
- [ ] Kindly make your issues link direct to the following link because right now it is taking you to some other repository's issues page 🤔

```
https://github.com/porag-m06/Porag-s-Awesome-Leaderboard/issues
```